angular/projects/spark-angular/src/lib/components/sprk-modal/sprk-modal.stories.ts

```diff
@@ -0,0 +1,92 @@
+import { storyWrapper } from '../../../../../../.storybook/helpers/storyWrapper';
+import { SprkModalModule } from './sprk-modal.module';
+
+export default {
+  title: 'Components|Modals',
+  decorators: [
+    storyWrapper(
+      storyContent => (
+        `<div class="sprk-o-Box">${ storyContent }<div>`
+      )
+    )
+  ],
+};
+
+const modules = {
+  imports: [
+    SprkModalModule,
+  ],
+};
+
+export const defaultStory = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-modal
+      [(isVisible)]="choiceVisible"
+      title="Are you sure?"
+      (hide)="toggleChoiceModal($event)"
+      modalType="choice"
+      idString="modal-choice-1"
+      (confirmClick)="confirmClick($event)"
+    >
+      This is some modal content.
+    </sprk-modal>
+  `,
+  props: {
+    choiceVisible: true,
+    toggleChoiceModal: (event: any) => {
+      event.preventDefault();
+      alert('cancel click');
```

Comment:
It feels unexpected to click the modal in storybook and get a browser alert saying "cancel click". Rob and I have chatted about this experience after we saw the react modal implemented that way but with things being so busy haven't been able to chat about it with everyone to get opinions. Should we bring this up now?

Reply:
Yeah, I thought that maybe was odd too. Happy to chat about it before we merge.